README.md

```diff
@@ -281,7 +281,15 @@ The server is using `hadoop-azure` to read Azure Data Lake Storage Gen2. We supp
 ```
 `YOUR-ACCOUNT-NAME` is your Azure storage account and `YOUR-ACCOUNT-KEY` is your account key.
 
-More cloud storage supports will be added in the future.
+### Google Cloud Storage
+
+We support using [Service Account](https://cloud.google.com/iam/docs/service-accounts) to read Google Cloud Storage. You can find more details in [GCP Authentication Doc](https://cloud.google.com/docs/authentication/getting-started).
+
+```
```

Comment:
To set up the credentials, you can specify the environment `GOOGLE_APPLICATION_CREDENTIALS` like this before starting the Delta Sharing Server.